### DISCUSSION TITLE
What's happened recently?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#239] [General]
There was a period you didn't make any commits. I also stopped checking for new commits on your repo for a while. I only revisit your repo recently and found you are making commits again.

**Comment by objeck** [comment]
Found a crazy syntax bug that took weeks to fix. It was not a major issue, more of a WTF language designer issue.

**Reply by ghost** [reply]
Are you sick or just busy with real life jobs? You are very inactive recently.

**Reply by objeck** [reply]
Busy with work

**Reply by ghost** [reply]
What do you think about making a new release? I could build from source to have the latest improvements, but for unknown reasons, obc will always generate a .obm file that you said shouldn't be generated at all (the present of this file indicated there is something wrong). This doesn't happen with the official release by you.

**Comment by objeck** [comment]
Working on that now. What toolchain are you using to build the code?

**Reply by ghost** [reply]
I always use MSYS2 as I'm on a legacy Windows version VS Studio no longer support. The obc built on MSYS2 either with UCRT64 or CLANG64 will always generate a .obm file.

**Comment by objeck** [comment]
I just built with MSYS2 UCRT64 and did not run into that issue. Are you adding ``-asm`` on the command line?

**Reply by ghost** [reply]
I missed one detail. I edited loader.cpp to allow running of .obe compiled by previous versions of obc. This maybe the reason.
